crates/oct-platform/README.md

```
@@ -8,3 +8,31 @@ export GITHUB_CLIENT_SECRET=<CLIENT_SECRET>
 
 cargo run -p oct-platform
 ```
+
+## SSR UI framework
+
+### Templates folders structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)